apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -1,24 +1,26 @@
-import { createFolderSchema, readFolderSchema } from "~/schemas/folder"
+import {
+  createFolderSchema,
+  editFolderSchema,
+  readFolderSchema,
+} from "~/schemas/folder"
 import { protectedProcedure, router } from "~/server/trpc"
 import { db } from "../database"
 
 export const folderRouter = router({
   create: protectedProcedure
     .input(createFolderSchema)
-    .mutation(
-      async ({ ctx, input: { folderTitle, parentFolderId, ...rest } }) => {
-        const folder = await db
-          .insertInto("Resource")
-          .values({
-            ...rest,
-            type: "Folder",
-            title: folderTitle,
-            parentId: parentFolderId ? String(parentFolderId) : null,
-          })
-          .executeTakeFirstOrThrow()
-        return { folderId: folder.insertId }
-      },
-    ),
+    .mutation(async ({ input: { folderTitle, parentFolderId, ...rest } }) => {
+      const folder = await db
+        .insertInto("Resource")
+        .values({
+          ...rest,
+          type: "Folder",
+          title: folderTitle,
+          parentId: parentFolderId ? String(parentFolderId) : null,
+        })
+        .executeTakeFirstOrThrow()
+      return { folderId: folder.insertId }
+    }),
```

Comment:
comment: seems to be formatting desync? does not look like there are changes. Why CI never catch :O

Reply:
done

Reply:
not sure what's the intended outcome here - can hint abit?